### PR TITLE
Access rules extentions

### PR DESCRIPTION
**Pull request by travlos**
Added support to get case rules
Fix in query for results
Use GroupId in query

### CODE REVIEW

File: src/Indice.Features.Cases.AspNetCore/Controllers/AdminAccessRulesController.cs [hunk 1]
```diff
@@ -38,6 +37,16 @@ public async Task<IActionResult> GetAccessRules([FromQuery] ListOptions<GetAcces
         return Ok(AccessRuless);
     }
 
+    /// <summary>Get Access rules for the specified case.</summary>
+    /// <param name="caseId"></param>
+    [HttpGet("/case/{caseId:Guid}")]
```
Comment (cleftheris) [inline src/Indice.Features.Cases.AspNetCore/Controllers/AdminAccessRulesController.cs:42]: This should be an extension to the case itself. So the correct path should follow /cases/{caseId}/access-rules

File: src/Indice.Features.Cases.AspNetCore/Services/AdminCaseService.cs [hunk 2]
```diff
@@ -89,15 +90,15 @@ public async Task<ResultSet<CasePartial>> GetCases(ClaimsPrincipal user, ListOpt
         // if client is systemic or admin, then bypass checks since no filtering is required.
         var isSystemOrAdmin = ((user.HasClaim(BasicClaimTypes.Scope, CasesApiConstants.Scope) && user.IsSystemClient()) || user.IsAdmin());
 
-
         var userId = user.FindSubjectIdOrClientId();
-        string inputGroupId = null;
+        string inputGroupId = user.GetUserGroup(_options.GroupIdClaimType);
```
Comment (cleftheris) [inline src/Indice.Features.Cases.AspNetCore/Services/AdminCaseService.cs:94]: Rename GetUserGroup to FindUserGroupId

File: test/Indice.Features.Cases.AspNetCore.Tests/AccessRulesServiceTest.cs [hunk 3]
```diff
@@ -83,6 +83,28 @@ await caseMembersService.AdminCreate(Admin(),
...
+        await caseMembersService.AdminCreate(Admin(),
+            new AddAccessRuleRequest() {
+                AccessLevel = 0,
+                MemberUserId = Guid.NewGuid().ToString(),
+                RuleCaseId = @case.Id
+            });
+        await caseMembersService.AdminCreate(Admin(),
+           new AddAccessRuleRequest() {
+               AccessLevel = 1,
+               MemberRole = "Admin",
```
Comment (cleftheris) [inline test/Indice.Features.Cases.AspNetCore.Tests/AccessRulesServiceTest.cs:100]: Use the constant `BasicRoleNames.Administrator`

File: src/Indice.Features.Cases.AspNetCore/Controllers/AdminAccessRulesController.cs (hunk 1 above)
Comment (travlos) [reply src/Indice.Features.Cases.AspNetCore/Controllers/AdminAccessRulesController.cs]: Updated

**Pull request merged**